tests/ha/haproxy.pm

```diff
@@ -0,0 +1,107 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test haproxy resource agent
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils 'zypper_call';
+use utils 'systemctl';
```

Comment:
No need to import `utils` twice just because you require to use 2 of its methods. You can pass a list with the methods you require in a single line, for example:

```
use utils qw(zypper_call systemctl);
```